mlx-mkbfb

```diff
@@ -66,7 +66,6 @@ image_list = (
     ("ddr-cert",  "DDR Images Content Certificate",        38),
     ("ddr_ini", "File From Which Will Load DDR (pseudo spd) Parameters", 32),
     ("snps_images", "Combined SNPS Images for all DIMM types", 33),
-    ("ddr5_snps_images", "DDR5 Combined SNPS Images for all DIMM types", 40),
```

Comment:
Same here. For backward compatibility we shouldn't get rid of this entry. Once users adapt the new image, then we can free this ID and remove the entry. What do you think?